package.json

```diff
@@ -19,43 +19,42 @@
     "ionic_webpack": "webpack.config.js"
   },
   "dependencies": {
```

Comment:
Many of the new dependency versions were copied from Ionic's changelog, and every new Ionic release seems to want us to upgrade blocks of dependencies at a time, so I'd actually argue that lines [22](https://github.com/umts/pvta-multiplatform/pull/413/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R22) through [38](https://github.com/umts/pvta-multiplatform/pull/413/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R38) and [50](https://github.com/umts/pvta-multiplatform/pull/413/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R50) through [52](https://github.com/umts/pvta-multiplatform/pull/413/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R50) stay un-`^`'d.